dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/features/ServerApi.java

```diff
@@ -150,6 +150,17 @@ String cloneServer(@PayloadParam("id") String id, @PayloadParam("imageName") Str
    @MapBinder(BindToJsonPayload.class)
    void startServer(@PayloadParam("id") String id);
 
+   @Named("server:cleanServer")
+   @POST
+   @Path("/cleanServer")
+   @Produces(MediaType.APPLICATION_JSON)
+   @MapBinder(BindToJsonPayload.class)
+   /**
+    * Operation for cleaning servers with FAILED_ADD state
+    * @see org.jclouds.dimensiondata.cloudcontrol.domain.State.FAILED_ADD
+    */
+   void cleanServer(@PayloadParam("id") String id);
```

Comment:
I have not added a live test for this particular operation as it would be difficult to setup in a live scenario.